src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -1035,7 +1041,13 @@ public static String getPreferredLanguage(@NonNull Map<String, Map<String, Objec
                                 }
                             } else if (StringUtils.isNumeric(userEntered)) { // separately process expressions with numbers
                                 NestedQueryBuilder query = addRangeLimitForNumber(
-                                        Filter.SMALLER_EQUALS, surveyParam[1], userEntered, finalQuery, queryPartsMap);
+                                        Filter.SMALLER_EQUALS, surveyParam[1].trim(), userEntered, finalQuery, queryPartsMap, ACTIVITIES_QUESTIONS_ANSWER_ANSWER);
+                                if (query != null) {
+                                    parentNestedOfRangeBuilderOfNumbers = query;
+                                }
```

Comment:
The same piece of code is presented on line 959, I think it will be good idea to extract this piece of code as a method to avoid code duplication.